docs/csharp/language-reference/keywords/snippets/usings.cs

```diff
@@ -0,0 +1,153 @@
+            ModernUsing();
+            TryFinallyExample();
+            MultipleUsings();
+            ModernMultipleUsings();
+            DeclareBeforeUsing();
+        }
+
+        private static void UsingExample()
+        {
+            // <SnippetFirstExample>
+            string manyLines=@"This is line one
+            This is line two
+            Here is line three
+            The penultimate line is line four
+            This is the final, fifth line.";
+
+            using (var reader = new StringReader(manyLines))
+            {
+                string? item;
+                do {
+                    item = reader.ReadLine();
+                    Console.WriteLine(item);
+                } while(item != null);
+            }
+            // </SnippetFirstExample>
+        }
+
+        private static void ModernUsing()
+        {
+            // <SnippetModernUsing>
+            string manyLines=@"This is line one
+            This is line two
+            Here is line three
+            The penultimate line is line four
+            This is the final, fifth line.";
+
+            using var reader = new StringReader(manyLines);
+            string? item;
+            do {
+                item = reader.ReadLine();
+                Console.WriteLine(item);
+            } while(item != null);
+            // </SnippetModernUsing>
+        }
+
+        private static void TryFinallyExample()
+        {
+            // <SnippetTryFinallyExample>
+            string manyLines=@"This is line one
+            This is line two
+            Here is line three
+            The penultimate line is line four
+            This is the final, fifth line.";
+
+            var reader = new StringReader(manyLines);
+            try {
+                string? item;
+                do {
+                    item = reader.ReadLine();
+                    Console.WriteLine(item);
+                } while(item != null);
+            } finally
+            {
+                reader?.Dispose();
+            }
+            // </SnippetTryFinallyExample>
+        }
+
+        private static void MultipleUsings()
+        {
+            // <SnippetMultipleUsings>
+            string numbers=@"One
+            Two
+            Three
+            Four.";
+            string letters=@"A
+            B
+            C
+            D.";
+
+            using (StringReader left = new StringReader(numbers),
+                right = new StringReader(letters))
+            {
+                string? item;
+                do {
+                    item = left.ReadLine();
+                    Console.Write(item);
+                    Console.Write("    ");
+                    item = right.ReadLine();
+                    Console.WriteLine(item);
+                } while(item != null);
+            }
+            // </SnippetMultipleUsings>
+        }
+
+        private static void ModernMultipleUsings()
+        {
+            // <SnippetModernMultipleUsings>
```

Comment:
This is introduced as multiple usings but it's actually one using that instantiates two objects.